src/colbert_beir/index_and_score.py

```diff
@@ -0,0 +1,68 @@
+from collections import defaultdict
+from pathlib import Path
+from colbert import Indexer, Searcher
+from colbert.infra import ColBERTConfig
+import ir_datasets
+from tqdm import tqdm
+from ranx import Run as ranx_run
+from ranx import Qrels, evaluate
+
+
+def build_colbert_index(
+    dataset_name: str,
+    model_name_or_path: str,
+    checkpoint_path: str,
+    collection: list[str],
+    tmp_path: str,
+):
+    config = ColBERTConfig(
+        nbits=8,
```

Comment:
Is this v2 ColBERT, with centroids and such?

Reply:
ColBERTv2+PLAID indexing for ease-of-use on the largest datasets yes. We could revert, to v1 style indexing but in my experience it doesn't significantly impact results

Reply:
We will be running most of our baselines anyways no?
Or we want to compare to results reported in papers? (you know I like IR reproducibility)

Reply:
> ColBERTv2+PLAID indexing for ease-of-use on the largest datasets yes.

Great, just making sure we're using the fastest option for eval :)